apps/health_campaign_field_worker_app/lib/pages/individual_details.dart

```diff
@@ -0,0 +1,135 @@
+                      } else {
+                        form.markAllAsTouched();
+                      }
+                    },
+                    child: Center(
+                      child: Text(
+                        AppLocalizations.of(context).translate(
+                            i18.individualDetails.submitButtonLabelText,),
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+            ),
+            children: [
+              DigitCard(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  mainAxisSize: MainAxisSize.min,
+                  children: [
+                    Text(
+                      i18.individualDetails.individualsDetailsLabelText,
+                      style: theme.textTheme.displayMedium,
+                    ),
+                    Column(children: [
+                      DigitTextFormField(
+                        formControlName: 'individualName',
+                        label: i18.individualDetails.nameLabelText,
+                      ),
+                      DigitCheckBox(
+                        checkBoxText: i18.individualDetails.checkboxLabelText,
+                        onChange: () {},
+                        isChecked: true,
+                      ),
+                      DigitTextFormField(
+                        formControlName: 'idType',
+                        label: i18.individualDetails.idTypeLabelText,
+                      ),
+                      Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          DigitTextFormField(
+                            formControlName: 'idNumber',
+                            label: i18.individualDetails.idNumberLabelText,
+                          ),
+                          const SizedBox(
+                            height: 4,
+                          ),
+                          Text(
+                            i18.individualDetails.idNumberSuggestionText,
+                            style: Theme.of(context).textTheme.bodySmall,
+                          ),
+                        ],
+                      ),
+                      DigitDobPicker(
+                        datePickerFormControl: 'dob',
+                        ageInputFormControl: 'age',
+                        onChangeDate: (DateTime? dateTime) {},
+                        datePickerLabel: i18.individualDetails.dobLabelText,
+                        ageFieldLabel: i18.individualDetails.ageLabelText,
+                        separatorLabel:
+                            i18.individualDetails.separatorLabelText,
+                      ),
+                      DigitDropDown(
+                        label: i18.individualDetails.genderLabelText,
+                        initialValue: 'Male',
+                        menuItems: const ['Male', 'Female'],
```

Comment:
Should have an 'Other' option.
List of values can be added to and read from field-app-config MDMS
Labels for the dropdown items should be localized

Reply:
Since this was only static portion of the code I have hardcoded these two values as placeholders for now. Widget is made in such a way that it uses data from an external source.

Reply:
yup let's add a config to the field-app-config MDMS configuration and update this screen to read the list from this config so this is complete and we don't have to revisit it for this later.